src/query/interface-manager/src/interface_mgr.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 Datafuse Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations
+
+use std::sync::Arc;
+
+use chrono::DateTime;
+use chrono::Utc;
+use common_base::base::GlobalInstance;
+use common_catalog::table_context::TableContext;
+use common_exception::Result;
+use common_storages_fuse::FuseTable;
+
+/// All `ee` interface need to add a fn in `InterfaceManager` trait
```

Comment:
Instead of a big interface which contains all the ee functions, I guess it'd better to seperate the different ee interfaces into different places to make each module more domain-specific.

Reply:
Agreed, it could be an effective way to ensure no config breaking changes occur